Makefile

```diff
@@ -44,6 +55,20 @@ rpm: dist-gzip welder-web.spec
 	find `pwd`/output -name '*.rpm' -printf '%f\n' -exec mv {} . \;
 	rm -r "`pwd`/rpmbuild"
 
+cockpit-composer-rpm: dist-gzip cockpit-composer.spec
+	mkdir -p "`pwd`/output"
+	mkdir -p "`pwd`/rpmbuild"
+	rpmbuild -bb \
```

Comment:
Can you specify `/usr/bin/rpmbuild` here? There's an issue with the Jenkins builders where sometimes they can't find programs in the standard path. Not sure why is that.